app/lib/validators/csv_file_validator.rb

```diff
@@ -3,7 +3,7 @@
 # CsvFileValidator
 class CsvFileValidator < ActiveModel::Validator
   def validate(record)
-    the_file = record.params[record.form_sym][:file]
+    the_file = record.params[record.form_sym]&.fetch(:file, nil)
```

Comment:
Same as before.  `&` protects the thing on the left being `nil`, not the thing on the right.